source/form.html

```diff
@@ -0,0 +1,184 @@
+<!DOCTYPE html>
+<html class="page" lang="ru">
+
+<head>
+  <meta charset="utf-8">
+  <title>HTML Academy: Кэт энерджи</title>
+</head>
+
+<body class="page__body">
+
+  <header class="page-header">
+    <a class="page-header__logo-cat">
+      <img src="https://via.placeholder.com/50x50" alt="Кот на логотипе «Cat Energy».">
+    </a>
+    <img class="page-header__logo" src="https://via.placeholder.com/100x50" alt="Логотип «Cat Energy».">
+    <nav class="main-nav">
+      <button class="main-nav__toggle" type="button">
+        <span class="visually-hidden">
+          Открыть меню.
+        </span>
+      </button>
+
+      <ul class="main-nav__list">
+        <li class="main-nav__list-item">
+          <a href="index.html">
+            Главная
+          </a>
+        </li>
+        <li class="main-nav__list-item">
+          <a href="catalog.html">
+            Каталог продукции
+          </a>
+        </li>
+        <li class="main-nav__list-item">
+          <a href="form.html">
```

Comment:
Мы находимся на этой странице, поэтому уже не надо человека куда-то направлять:
```suggestion
          <a>
```